planetary-social/nos
# Show authors on Discover tab instead of notes

## Issues
#1022 
#1023
#1024 

Changes layout of Discover tab to feature authors instead of notes. Adds new authors and categories to Discover tab for testing.

## How to Test
1. Navigate to Discover tab
2. Observe new categories at the top
3. Observe authors below
4. Scroll to view authors
5. Tap follow button to follow and unfollow authors
6. Tap a category to filter by that category
7. Tap an author to view their profile

## Screenshots
| All | Tech |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 Pro - 2024-04-24 at 14 58 28](https://github.com/planetary-social/nos/assets/59564/f29d0532-a4fc-4104-873e-da2e4b762d28) | ![Simulator Screenshot - iPhone 15 Pro - 2024-04-24 at 14 58 31](https://github.com/planetary-social/nos/assets/59564/6c69f048-04d0-4a3f-a62c-47b01a86b7e2) |
